packages/liquidator/index.js

```diff
@@ -17,12 +19,14 @@ const {
 
 // Contract ABIs and network Addresses
 const { getAbi, getAddress } = require("@umaprotocol/core/index");
+const { ONE } = require("long");
```

Comment:
What's this used for and what is `long`?

Reply:
Probably some auto formating thing

Reply:
ok pls remove